WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/BlockEditorPage.kt

```diff
@@ -50,6 +50,11 @@ class BlockEditorPage {
         return this
     }
 
+    fun dismissContentStructure(): BlockEditorPage {
+        Espresso.pressBack()
+        return this
+    }
```

Comment:
Have you considered including the content structure dismissal as part of `verifyContentStructure(blocks: Int, words: Int, characters: Int)`? It makes sense to me as ideally the assertion step should not "break" the test flow, like having to always call a `dismiss` after a `verify`. WDYT?

Reply:
Nice suggestion, yeah I think it should dismiss the dialog after asserting, I've applied the changes in https://github.com/wordpress-mobile/WordPress-Android/pull/18705/commits/2abfcda45744af4eb720e087d474602f5d00e835